Source/Dafny/Compilers/Compiler-go.cs

```diff
@@ -196,7 +202,11 @@ private struct Import {
       importWriter.WriteLine("{0} \"{1}\"", id, path);
 
       if (!import.SuppressDummy) {
-        importDummyWriter.WriteLine("var _ {0}.{1}", id, DummyTypeName);
+        if (id == "os") {
+          importDummyWriter.WriteLine("var _ = os.Args");
+        } else {
+          importDummyWriter.WriteLine("var _ {0}.{1}", id, DummyTypeName);
+        }
```

Comment:
This looks odd — what happens if someone has a Dafny module called `os`?

Reply:
Then there will be a compilation failure. What would be the alternative?

Reply:
A clean error produced by Dafny; but what is bothering me is the special case on the string `os`.  We could store tuples in the list of modules (maybe `("os", "Args")` and `(id, DummyTypeName)`)

Reply:
What do you mean by "we could store tuples in the list of modules"?
One could make this error go away by suppressing these dummy calls.
But overall, I think that if this is a problem down the road, the Go compiler will have to compile a Dafny module named "os" to something else, since "os" is the base package in Go anyway.